.github/workflows/docs.yml

```diff
@@ -37,6 +37,13 @@ jobs:
         shell: bash
         run: |
           tox -edocs
+      - name: Upload docs artifact
+        if: always()
```

Comment:
Any reason you like the if statement here?

Reply:
I think we can just remove the if statement here

Reply:
I see from @garrison comment below, the if: always is in place to force a broken build to upload